whisper/whisperv6/whisper_test.go

```diff
@@ -368,49 +368,49 @@ func TestWhisperSymKeyManagement(t *testing.T) {
 		t.Fatalf("failed w.GetSymKey(id1): false positive.")
 	}
 	if k1 != nil {
-		t.Fatalf("failed GetSymKey(id1): false positive.")
+		t.Fatalf("failed GetSymKey(id1): false positive. key=%v", k1)
 	}
 	k2, err = w.GetSymKey(id2)
 	if err != nil {
-		t.Fatalf("failed w.GetSymKey(id2).")
+		t.Fatalf("failed w.GetSymKey(id2). err=%v", err)
 	}
 	if w.HasSymKey(id1) {
-		t.Fatalf("failed to delete first key: still exist.")
+		t.Fatal("failed to delete first key: still exist.")
 	}
 	if !w.HasSymKey(id2) {
-		t.Fatalf("failed to delete first key: second key does not exist.")
+		t.Fatal("failed to delete first key: second key does not exist.")
 	}
-	if k1 != nil {
-		t.Fatalf("failed to delete first key.")
+	if k1 != nil { // TODO isn't this an ineffectual check? if k1 were not nil, that would've been caught on L370
```

Comment:
@gballet can I remove this check? It seems ineffectual.

Reply:
yes, it's probably some merge error